controllers/mover/restic/restic_test.go

```diff
@@ -93,6 +93,60 @@ var _ = Describe("Restic retain policy", func() {
 	})
 })
 
+var _ = Describe("Restic unlock", func() {
+	var m *Mover
+	var owner *corev1.ConfigMap
+	logger := zap.New(zap.UseDevMode(true), zap.WriteTo(GinkgoWriter))
+	var start metav1.Time
+
+	BeforeEach(func() {
+		start = metav1.Now()
+		// The underlying type of owner doesn't matter
+		owner = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:              "name",
+				Namespace:         "ns",
+				CreationTimestamp: start,
+			},
+		}
+		m = &Mover{
+			logger:       logger,
+			owner:        owner,
+			sourceStatus: &volsyncv1alpha1.ReplicationSourceResticStatus{},
+		}
+	})
+
+	When("Unlock is not set in the spec", func() {
+		It("shouldUnlock() should return false", func() {
+			Expect(m.shouldUnlock()).To(BeFalse())
+		})
+	})
```

Comment:
Do you want to add a test case for `.spec` is "", but `.status` has a value?

I see it's covered in the job tests below.